### ISSUE TITLE
Input padding in MeshConv_transpose

### ISSUE THREAD

**Issue by AndreLitta** [maxjiang93/ugscnn#6]
Hi, 
thank you for the great work and for sharing it.

I'm referring to line 90, within ugscnn/meshcnn/ops.py, in MeshConv_transpose:
	# pad input with zeros up to next mesh resolution
	ones_pad = torch.ones(*input.size()[:2], self.nv_pad).to(input.device)
	input = torch.cat((input, ones_pad), dim=-1)

doesn't padding the input with a constant (being it zeros or ones) result 
in a bad input to the MeshConv, as the gradients will be artificially large due 
to real data being alternated with this arbitrary constant? 

Will this in turn restrict the ability of the MeshConv_transpose to properly map the 
low resolution data to high resolution, especially when the output of the architecture 
is at an higher level than the input, thus not having the skip connection available
as an input to the Up class in ugscnn/experiments/exp4_sphere_climate/model.py?

Is my reasoning in this case correct?

Thank you in advance for your help.
